pkg/controllers/state/cluster.go

```diff
@@ -82,6 +82,16 @@ func NewCluster(clk clock.Clock, client client.Client) *Cluster {
 //
 //nolint:gocyclo
 func (c *Cluster) Synced(ctx context.Context) (synced bool) {
+	start := time.Now()
+	ClusterStateUnsyncedTimeSeconds.With(map[string]string{}).Set(time.Since(start).Seconds())
```

Comment:
Why do you need to set this metric twice?

Reply:
Agreed. Not needed.